h2o-bindings/bin/gen_java.py

```diff
@@ -637,12 +669,16 @@ def generate_main_class(endpoints):
             try {
               int i = 0;
               for (String id: ids) {
-                keys[i] = clz.newInstance();
+                keys[i] = clz.getConstructor().newInstance();
```

Comment:
regarding `getConstructor` - neat, good to know! nice catch!

regarding `clz` - just wanted to be sure, that we are not depending on some explicit class loader. But it seems that we are ok.